filters/resources.txt

```diff
@@ -2310,3 +2310,34 @@ damoh-defuser.js application/javascript
     var observer = new MutationObserver(cleanVideoAsync);
     observer.observe(document.documentElement, { childList: true, subtree: true });
 })();
+
+
+twitch-videoad.js application/javascript
+(function() {
+	var ourMediaPlayer = undefined;
+	Object.defineProperty(window, 'MediaPlayer', {
+		set: function(newMediaPlayer) {
+			if (typeof(ourMediaPlayer) !== 'undefined')
+				return;
+			var oldLoad = newMediaPlayer.MediaPlayer.prototype.load;
```

Comment:
This might break the site if Twitch removes the `MediaPlayer.MediaPlayer.prototype.load` property. So another try-catch around this and the following statement might be in order.